gcloud-java-core/src/main/java/com/google/gcloud/RetryParams.java

```diff
@@ -62,6 +66,9 @@
   private final double retryDelayBackoffFactor;
   private final long totalRetryPeriodMillis;
 
+  // Some services may have different backoff requirements listed in their SLAs. Be sure to override
```

Comment:
I would move that comment to ServiceOptions.defaultRetryParams (you can *also* keep it here if you want).